app/views/claims/pages/terms.en.html.erb

```diff
@@ -38,7 +38,7 @@
         </p>
 
         <p class="govuk-body">
-          You can contact us using the following email address: <%= govuk_mail_to t("claims.support_email") %>
+          You can contact us using the following email address: <%= govuk_mail_to t("claims.support_email_html") %>
```

Comment:
This doesn't work. The display and mailto address need to be distinct, unfortunately.

There are a couple more like this in the PR.

Reply:
Ah yes, you're right! I tried playing with the `govuk_mail_to` method but it doesn't allow me to use it in the same way as the `mail_to` method does, I meant to switch all of these back to the original but I've clearly missed some, nice spot!